src/setup/node.rs

```diff
@@ -67,11 +67,35 @@ impl Node {
         let config = NodeConfig::new();
         let meta = NodeMetaData::new()?;
 
-        Ok(Self {
+        let mut node = Self {
             config,
             meta,
             process: None,
-        })
+        };
+
+        // insert the config file cmd args
+        match node.meta.kind {
+            NodeKind::Zebra => {
+                // Bit more convoluted since we need to insert the args before `start`, which comes last.
+                let start_arg = node
+                    .meta
+                    .start_args
+                    .pop()
```

Comment:
You could probably make an `insert` at `i = len - 2` instead?

Reply:
I was worried that having it twice in a row might be confusing (since its two separate args). But I think yours is still better.